src/main/java/io/federecio/dropwizard/swagger/SwaggerBundleConfiguration.java

```diff
@@ -15,6 +15,7 @@
  */
 package io.federecio.dropwizard.swagger;
 
+import org.eclipse.jetty.util.URIUtil;
```

Comment:
UriBuilder don't have constants for HTTP scheme's. That's why picked jetty URIUtil. 
It wouldn't be a problem as dropwizard uses jetty by default.